doc/Language/faq.pod6

```diff
@@ -370,6 +370,99 @@ L<.comb|/routine/comb> is accepting an optional L<Int|/type/Int>:
 .say for 'abcdefghijklmnopqrstuvwxyz'.comb: 8;  # OUTPUT: «abcdefgh␤ijklmnop␤qrstuvwx␤yz»
 =end code
 
+=head2 List: How can I remove duplicates items in a list
+
+Use L<.unique|/routine/unique>:
+
+=begin code
+(1, 2, 3, 2, 3, 1, 1, 0).unique;  # OUTPUT: «(1 2 3 0)␤»
+=end code
+
+=head2 List: How can I delete an element from a list
```

Comment:
As above, if you want people to find it, just googling it takes you to [this stackoverflow question](https://stackoverflow.com/questions/60404000/delete-an-element-from-a-list-or-array-without-the-any-artefact-in-raku), which is good enough, and besides, provides another way of doing the same, using `:delete`

Reply:
Removed